battleShip.py

```diff
@@ -0,0 +1,280 @@
+import random
+import pickle
+
+
+class Ship:
```

Comment:
can we use a named tuple if it is a storage class only?

Reply:
Yes